includes/admin/stripe-settings.php

```diff
@@ -3,6 +3,33 @@
 	exit;
 }
 
+if ( wc_stripe()->connect->is_connected() ) {
+	$reset_link = add_query_arg(
+		array( 'reset_stripe_api_credentials' => true ),
+		admin_url( 'admin.php?page=wc-settings&tab=checkout&section=stripe' )
+	);
+
+	$api_credentials_text = sprintf(
+		__( '%1$sReset Stripe account keys%2$s', 'woocommerce-gateway-stripe' ),
+		'<a href="' . $reset_link . '" class="button button-secondary">',
+		'</a>'
+	);
+} else {
+	$oauth_url = wc_stripe()->connect->get_oauth_url();
+
+	if ( ! is_wp_error( $oauth_url ) ) {
+		$api_credentials_text = sprintf(
+			__( '%1$sSetup or link an existing Stripe Account%2$s By clicking this button you agree to the %3$sTerms of Service%2$s. Or manually enter Stripe keys below.', 'woocommerce-gateway-stripe' ),
+			'<a href="' . $oauth_url . '" class="button button-primary">',
+			'</a>',
+			'<a href="https://wordpress.com/tos">'
+
+		);
+	} else {
+		$api_credentials_text = __( 'Manually enter Stripe keys below.', 'woocommerce-gateway-stripe' );
```

Comment:
Please replace Stripe keys with Stripe account keys for clarity.

Reply:
https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1267/commits/f9ad9f38bba43311f5d8998705e2b71325202034